Multisig/UI/App/ViewControllerFactory.swift

```diff
@@ -84,11 +84,11 @@ enum ViewControllerFactory {
             UIBarButtonItem(barButtonSystemItem: .close, target: vc, action: #selector(CloseModal.closeModal))
     }
 
-    static func removeNavigationItem(_ vc: UIViewController) {
+    static func makeTransparentNavigationBar(_ vc: UIViewController) {
         // remove underline from navigationItem
         let navigationBarAppearance = UINavigationBarAppearance()
         navigationBarAppearance.configureWithTransparentBackground()
-        navigationBarAppearance.backgroundColor = .clear
+        navigationBarAppearance.backgroundColor = .backgroundSecondary
```

Comment:
Not sure if this will work fine for all our use cases

Reply:
Previously in dark mode the navigation bar becomes transparent